api/v1alpha/subschema/sauce.schema.json

```diff
@@ -56,6 +56,11 @@
             "id"
           ],
           "additionalProperties": false
+        },
+        "retries": {
+          "description": "The number of times to retry a failing suite.",
+          "type": "integer",
+          "minimum": 1
```

Comment:
Hm, isn't that ought out to be `0`? Since we default to 0 retries 🤔

Reply:
oops, you're right.